src/components/Zodiac/Zodiac.js

```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+import './Zodiac.css';
+export default function Zodiac(props) {
+  return (
+    <div className='zodiac'>
+      <img src={`${process.env.PUBLIC_URL}/zodiacs/${props.name}.png`} />
+      <h2 className='sign'>{props.name}</h2>
+      <p className='dates'>{props.dates}</p>
+    </div>
+  );
+}
```

Comment:
Here is where we can utilize destructuring in our props argument to pull in the specific properties, something that we will be learning soon :)